test/sanity/window-showdevtools/index.html

```diff
@@ -0,0 +1,22 @@
+<html>
```

Comment:
NW.js requires HTML documents
```suggestion
<!DOCTYPE html>
<html>
```